src/main/java/com/ldtteam/structurize/config/ServerConfiguration.java

```diff
@@ -59,6 +62,7 @@ protected ServerConfiguration(final ForgeConfigSpec.Builder builder)
         maxCachedSchematics = defineInteger(builder, "maxCachedSchematics", 100, 0, 100000);
         maxBlocksChecked = defineInteger(builder, "maxBlocksChecked", 1000, 0, 100000);
         schematicBlockLimit = defineInteger(builder, "schematicBlockLimit", 100000, 1000, 1000000);
+        iteratorType = defineString(builder, "iteratorType", "default");
```

Comment:
Can this have validator, something like `str.lowercase() in possibleValues`?

Reply:
Where?

Reply:
Perhaps it should be an EnumValue instead?

Reply:
Yes, the main reason it's not an Enum is that a) Other mods can add iterators and b) We can add for testing purposes on the MineColonies side iterators easily without having to recompile structurize etc.

Reply:
we can still emulate enum in the way vanilla does it with eg. RenderType

Reply:
I don't really think that is worth it.